mreg_cli/commands/policy.py

```diff
@@ -5,13 +5,14 @@
 import argparse
 from typing import Any
 
+from mreg_cli.api.history import HistoryResource
+from mreg_cli.api.history import HistoryResource
```

Comment:
Duplicate?

Reply:
Some import auto sorting weirdness when selecting ranges to commit...